### PR TITLE
New module time and panel time, it's also an added a menu item to sho…

### PR DESCRIPTION
**Pull request by sondy99**
…w/hide the time controls; when you press play every change will be undone when you press stop

### CODE REVIEW

File: Source/ModuleTime.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+ModuleTime::ModuleTime()
+{
+}
+
+ModuleTime::~ModuleTime()
+{
+}
+
+bool ModuleTime::Init(JSON* json) {
```
Comment (AlbertVVila) [inline Source/ModuleTime.cpp]: IF statement coding style :(

File: Source/ModuleTime.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	gameDeltaTime = 0.0f;
+	frameTimer.Start();
+	fpsTimer.Start();
+	return true;
+}
+
+update_status ModuleTime::Update(float dt) {
```
Comment (AlbertVVila) [inline Source/ModuleTime.cpp]: style :( 
(not gonna comment it more but several statements should change {} to match coding style)

File: Source/ModuleTime.cpp (hunk 1 above)
Comment (sondy99) [reply Source/ModuleTime.cpp]: Solved, the copy paste will kill me haha :)

File: Source/ModuleTime.cpp (hunk 2 above)
Comment (sondy99) [reply Source/ModuleTime.cpp]: Solved, the copy paste will kill me haha :)

**Comment by AlbertVVila** [comment]
Is temporaryScene.json removed on stop game? Should it be removed?

**Comment by sondy99** [comment]
> Is temporaryScene.json removed on stop game? Should it be removed?

Ufff!! sounds like a good idea, but this type of "temporary files" tends to be overwritten instead of delete and create new one every time.. but to be honest, I'm not sure which option has better general performance

**Pull request merged**